src/networkProviders/proxyNetworkProvider.ts

```diff
@@ -65,40 +65,57 @@ export class ProxyNetworkProvider implements INetworkProvider {
         return accountGuardian;
     }
 
-    async getFungibleTokensOfAccount(address: IAddress, _pagination?: IPagination): Promise<FungibleTokenOfAccountOnNetwork[]> {
+    async getFungibleTokensOfAccount(
+        address: IAddress,
+        _pagination?: IPagination,
+    ): Promise<FungibleTokenOfAccountOnNetwork[]> {
         let url = `address/${address.bech32()}/esdt`;
         let response = await this.doGetGeneric(url);
         let responseItems: any[] = Object.values(response.esdts);
         // Skip NFTs / SFTs.
-        let responseItemsFiltered = responseItems.filter(item => !item.nonce);
-        let tokens = responseItemsFiltered.map(item => FungibleTokenOfAccountOnNetwork.fromHttpResponse(item));
+        let responseItemsFiltered = responseItems.filter((item) => !item.nonce);
+        let tokens = responseItemsFiltered.map((item) => FungibleTokenOfAccountOnNetwork.fromHttpResponse(item));
 
         // TODO: Fix sorting
         tokens.sort((a, b) => a.identifier.localeCompare(b.identifier));
         return tokens;
     }
 
-    async getNonFungibleTokensOfAccount(address: IAddress, _pagination?: IPagination): Promise<NonFungibleTokenOfAccountOnNetwork[]> {
+    async getNonFungibleTokensOfAccount(
+        address: IAddress,
+        _pagination?: IPagination,
+    ): Promise<NonFungibleTokenOfAccountOnNetwork[]> {
         let url = `address/${address.bech32()}/esdt`;
         let response = await this.doGetGeneric(url);
         let responseItems: any[] = Object.values(response.esdts);
         // Skip fungible tokens.
-        let responseItemsFiltered = responseItems.filter(item => item.nonce >= 0);
-        let tokens = responseItemsFiltered.map(item => NonFungibleTokenOfAccountOnNetwork.fromProxyHttpResponse(item));
+        let responseItemsFiltered = responseItems.filter((item) => item.nonce >= 0);
+        let tokens = responseItemsFiltered.map((item) =>
```

Comment:
No reason for any of these variables to be let if no of them get reassigned. Applicable to the rest of the file too.

Reply:
done

Reply:
Awesome. 👍